markdown/ranger/ranger-kerberos.html.md.erb

```diff
@@ -0,0 +1,209 @@
+
+4. Start the Ranger Admin UI in a supported web browser. The default URL is \<ranger-admin-node\>:6080. 
+
+5. Locate the HAWQ service definition and press the **Edit** button. 
+
+6. Update the applicable **Config Properties** fields:
+
+    **HAWQ User Name*** - The HAWQ Ranger lookup role you identified or created in Step 2 above.  
+    **HAWQ User Password*** - The password you assigned to the HAWQ Ranger lookup role principal.  
+    **HAWQ Kerberos Service Name** - `postgres`  
+
+7. Click the **Test Connection** button. If you connect successfully, Ranger displays a dialog with the message:
+
+    ``` pre
+    Connected Successfully.
+    ```
+
+    You successfully configured Ranger to use Kerberos authentication when looking up HAWQ resource names.
+
+8. **Save** your changes to permanently configure Kerberos authentication for HAWQ Ranger resource lookup.
+
+
+## Configure HAWQ for Kerberized Ranger<a id="rps2ra_kerb_cfg"></a>
+
+When Ranger Authorization is enabled for HAWQ, the Ranger Plug-in Service fetches policies from the Ranger Administration host. When Kerberos is enabled for Ranger, you must specifically configure the Ranger Plug-in Service for this communication.
+
+This procedure includes:
+
+- Adding an administrative Ranger user named `postgres`
+- Updating HAWQ Ranger Plug-in Service configuration to utilize Kerberos
+- Updating the HAWQ service definition in the Ranger Admin UI
+
+
+### Procedure <a id="kerb_rps2ra_proc"></a>
+
+Perform the following steps to enable Ranger Plug-in Service policy refresh from a kerberized Ranger:
+
+1. Create a `postgres` Ranger administrative user.
+
+    1. Open the Ranger Admin UI. The default URL is \<ranger-admin-node\>:6080.
+    2. Navigate to **Settings > Users/Groups**.
+    3. Press the **Add New User** button.
+    4. Fill out the required fields in the **User Detail** dialog:
+
+        **User Name*** - `postgres`  
+        **New Password*** - The user's password  ?? IS THIS SIGNIFICANT ??  
+        **Password Confirm*** - Enter the password again  
+        **First Name*** - The first name of the user  
+        **Select Role*** - Select the **Admin** role
+
+2. Update the HAWQ Ranger Plug-in Service configuration to use Kerberos authentication when communicating with Ranger.
+
+    2. Edit the HAWQ Ranger Plug-in Service `rps.properties` configuration file and set the `RPS_AUTH` property to specify Kerberos authentication.
+
+        ``` shell
+        gpadmin@master$ vi $GPHOME/ranger/etc/rps.properties
+        ```
+    
+        ``` pre
+        RPS_AUTH=kerberos
+        ``` 
+
+    3. `rps.properties` also includes `RPS_PRINCIPAL` and `RPS_KEYTAB` properties. If unset, the Ranger Plug-in Service uses the Kerberos credentials cache in place for the HAWQ runtime user account (typically `gpadmin`). Set these properties if the default values are not appropriate for your environment.
```

Comment:
also includes **the**

If **these properties are** unset, the Ranger Plug-in Service uses the **cached Kerberos credentials that are available** for the HAWQ runtime user account (typically `gpadmin`).  <--- I *think* this edit is correct.